plugins/woocommerce/src/Blocks/BlockTypes/Checkout.php

```diff
@@ -354,6 +354,7 @@ protected function enqueue_data( array $attributes = [] ) {
 		$this->asset_data_registry->add( 'displayCartPricesIncludingTax', 'incl' === get_option( 'woocommerce_tax_display_cart' ) );
 		$this->asset_data_registry->add( 'displayItemizedTaxes', 'itemized' === get_option( 'woocommerce_tax_total_display' ) );
 		$this->asset_data_registry->add( 'forcedBillingAddress', 'billing_only' === get_option( 'woocommerce_ship_to_destination' ) );
+		$this->asset_data_registry->add( 'generatePassword', filter_var( get_option( 'woocommerce_registration_generate_password' ), FILTER_VALIDATE_BOOLEAN ) );
```

Comment:
If we move the password field to CheckoutFields, this setting wouldn't need to be passed frontend, you can just conditionally register that field there (or remove it).